src/chart/AccessibilityManager.tsx

```diff
@@ -0,0 +1,98 @@
+import { LayoutType } from '../util/types';
+
+interface ContainerOffset {
+  top: number;
+  left: number;
+}
+
+/* eslint-disable no-underscore-dangle */
+export class AccessibilityManager {
+  private _coordinateList: any[] = [];
+
+  private _activeIndex = 0;
+
+  private _mouseHandlerCallback: any;
+
+  private _container: any;
```

Comment:
I know these are coming from generateCategoricalChart which has terrible typing but if we could type them any more specifically that would be awesome

Reply:
Agree, the goal should be: No any. Leading by example that would slowly help us pave the way to a better future :D